com.unity.render-pipelines.universal/CHANGELOG.md

```diff
@@ -8,15 +8,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
 - Added [DisallowMultipleRendererFeature] attribute for Renderer Features.
+- Added a DepthNormalOnly pass for UniversalUnlit Shadergraph shaders.
+
+### Changed
+- MaterialReimporter.ReimportAllMaterials now batches the asset database changes to improve performance.
 
 ### Fixed
 - Fixed ShaderGraph needing updated normals for ShadowCaster in URP.
 - Fixed memory leak with XR combined occlusion meshes. [case 1366173]
 - Fixed an issue with MSAA falling back to the incorrect value when sample count 2 is not supported on some Android GPUs.
 
-### Changed
-- MaterialReimporter.ReimportAllMaterials now batches the asset database changes to improve performance.
-
```

Comment:
I moved this up, as our structure should be `Added -> Changed -> Fixed` in the changelogs